fileio/ft_filetype.m

```diff
@@ -1337,6 +1337,10 @@
   type = 'nihonkohden_m00';
   manufacturer = 'Nihon Kohden';
   content = 'continuous EEG';
+elseif filetype_check_extension(filename, '.EEG')
```

Comment:
there are many more file formats that end with *.eeg or *.EEG, so this is a very poor test. Please look at whether the first N bytes of the (binary?) file are unique.